x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList/index.scss

```diff
@@ -0,0 +1,7 @@
+.apmServiceList__serviceNameTooltip {
+  width: 100%;
+  .apmServiceList__serviceNameContainer {
+    // removes 24px referent to the icon placed on the left side of the text.
+    width: calc(100% - 24px);
+  }
+}
```

Comment:
I'm not sure if we should introduce sass since everything else is in styled-components and the future plans is to migrate away from sass (to another css-in-js approach).

Reply:
I just added a sass file because the `EuiTooltip` accepts and `anchorClassName` out of the box, so I wouldn't have to overwrite the component. But I can also do it NP.